tensorbay/client/segment.py

```diff
@@ -206,9 +207,18 @@ def _upload_file(self, data: FileMixin) -> None:
             )
 
             self._put_binary_file_to_azure(url, local_path, post_data)
+        elif backend_type == "fps":
+            if not host.startswith("http"):
+                host = f"http://{host}"
```

Comment:
When dose the host not start with `http`?

Reply:
If it is not given while creating local storage config.

Reply:
I don't understand `while creating local storage config`
why we will create config in `upload_file` process?